pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>JAVA_REST_API</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+            </resource>
+        </resources>
+    </build>
+
+    <properties>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <httpclient.version>4.5.14</httpclient.version>
+        <junit.version>5.9.1</junit.version>
+        <jackson.version>2.17.1</jackson.version>
+        <lombok.version>1.18.32</lombok.version>
+        <logback-classic.version>1.2.9</logback-classic.version>
+
+        <!--API Properties-->
+        <api.hostname>localhost</api.hostname>
+        <api.port>49000</api.port>
+        <api.scheme>http</api.scheme>
+        <api.username>0oa157tvtugfFXEhU4x7</api.username>
+        <api.password>X7eBCXqlFC7x-mjxG5H91IRv_Bqe1oq7ZwXNA8aq</api.password>
+        <api.tokenUrl>http://localhost:49000/oauth/token</api.tokenUrl>
```

Comment:
To be honest there is no need to save values in pom file. You can save them in config.properties file. It will be a more common way.